packages/patterns-react/src/patterns/LeadSpace/__stories__/LeadSpace.stories.js

```diff
@@ -33,16 +34,53 @@ if (DDS_LEADSPACE) {
         productive: 'productive',
       };
 
+      /**
+       * Return icon component based on the parameter
+       *
+       * @param {string} icon icon name parameter
+       * @returns {*} Icon JSX component or null
+       */
+      const selectIcon = icon => {
+        switch (icon) {
+          case 'ArrowRight':
+            return ArrowRight20;
+          case 'ArrowDown':
+            return ArrowDown20;
+          case 'Pdf':
+            return Pdf20;
+          default:
+            return null;
+        }
+      };
+
+      const availableIcons = {
+        ArrowRight: 'ArrowRight',
+        ArrowDown: 'ArrowDown',
+        Pdf: 'Pdf',
+        None: 'None',
+      };
+
+      const button1Icon = select(
+        'Button1 Icon',
+        availableIcons,
+        availableIcons.ArrowRight
+      );
+      const button2Icon = select(
+        'Button2 Icon',
+        availableIcons,
+        availableIcons.ArrowDown
+      );
+
       const buttons = [
         {
-          link: '',
-          copy: 'Primary action button',
-          renderIcon: 'ArrowDown',
+          link: text('Button2 link:', ''),
+          copy: text('Button2 label:', 'Primary action'),
+          renderIcon: selectIcon(button2Icon),
```

Comment:
Hmmm I don't think we can have these fields and the ` buttons={object('buttons', buttons)}` at the same time - notice if you modify in your text field it doesn't actually change

Reply:
I've changed these in the latest commit, now the only error we are getting is from the buttongroup component, one as the page loads in, and one as we change the icon to none, this is also present in the buttongroup story, so I don't know exactly how to proceed

Reply:
what errors are you getting? I'm not seeing any

Also noticed you removed the button object knob altogether - if that's the case can we put in fields so that users can change the button text?

Reply:
what errors are you getting? I'm not seeing any

Also noticed you removed the button object knob altogether - if that's the case can we put in fields so that users can change the button text?

Reply:
what errors are you getting? I'm not seeing any

Also noticed you removed the button object knob altogether - if that's the case can we put in fields so that users can change the button text?